lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -88,6 +88,22 @@ class BlockedToMMA : public mlir::RewritePattern {
   matchAndRewrite(mlir::Operation *op,
                   mlir::PatternRewriter &rewriter) const override {
     auto dotOp = cast<triton::DotOp>(op);
+
+    // operands
+    Value a = dotOp.getA();
+    Value b = dotOp.getB();
+    auto oldAType = a.getType().cast<RankedTensorType>();
+    auto oldBType = b.getType().cast<RankedTensorType>();
+
+    if (computeCapability < 70) {
```

Comment:
`v100` has a compute capability of 70 so changed it from `<=` to `<` because it supports MMA Layout.

Reply:
the error should be somewhere else. Probably in `semantic.dot` at the frontend. Here an assertion that `computeCapability >= 70` should be enough.

Reply:
Also, your check doesn't cover float8

Reply:
@ptillet Done as you have suggested :+1: